### PR TITLE
Fix get user and update user

### PR DESCRIPTION
**Pull request by cliarie**

### CODE REVIEW

File: code/student/user.py [hunk 1]
```diff
@@ -32,35 +32,43 @@ def __init__(self, uid, name, email, linkedin, degree, majors, minors, gpa, year
...
-dynamo_table = 'infra-resume-book-users'
+client = boto3.client('dynamodb', region_name='us-east-2')
+dynamo_table = "infra-resume-book-users"
 
 def get_user(id: int) -> str | None:
     response = client.get_item(
         TableName=dynamo_table,
         Key={
-            'id': id
+            "uin": id
```
Comment (devksingh4) [inline code/student/user.py:42]: If this is university UIN, we are not permitted to ask for this number (and we shouldn't have it anyway).

File: test.json [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
     "headers": {
         "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
         "Accept-Encoding": "gzip, deflate, sdch",
         "Accept-Language": "en-US,en;q=0.8"
     },
     "queryStringParameters": {
         "param1": "value1",
-        "param2": "value2"
+        "param2": "value2", 
+        "uid": {"N": "671901702"}
```
Comment (devksingh4) [inline test.json:13]: Do not use UIN. Use NetID. UIN should be treated as "secret-ish" per uni policy and we are not supposed to ask for UIN.

Comment (devksingh4) [reply test.json:13]: We also have no reason to know their UIN.

File: test.json [hunk 3]
```diff
@@ -1,18 +1,33 @@
 {
     "resource": "/myresource",
-    "path": "/api/v1/healthz",
-    "httpMethod": "GET",
+    "path": "/api/v1/student",
+    "httpMethod": "PUT",
```
Comment (devksingh4) [inline test.json:4]: Make sure to update swagger definition so API Gateway will deploy.

**Review by devksingh4 — CHANGES_REQUESTED** [review]
See comments. Additionally Josh may have more but this is my pass at it.

File: code/student/user.py (hunk 1 above)
Comment (Jish2) [reply code/student/user.py:42]: this refers to object_id in aad

File: test.json (hunk 2 above)
Comment (Jish2) [reply test.json:13]: in the process of updating dynamo

**Pull request merged**